src/main/java/seedu/address/logic/commands/ArchiveCommand.java

```diff
@@ -76,29 +66,35 @@ private Path initArchiveFilePath() {
      * @return Name for the directory for the archived file
      */
     private String initArchiveDirectoryPathName() {
-        return ARCHIVE_DIRECTORY_NAME.format(date);
+        Date date = new Date();
+        return ARCHIVE_DIRECTORY_NAME_FORMAT.format(date);
     }
 
     /**
      * Initialises the name for the archived file using the current date and time.
      * @return Name for the archived file
      */
     private String initArchiveFilePathName() {
-        return ARCHIVE_FILE_NAME.format(date) + ".json";
+        Date date = new Date();
+        return ARCHIVE_FILE_NAME_FORMAT.format(date) + ".json";
     }
 
     /**
      * Creates the archived address book file in the specified path and copies the information in the address book
```

Comment:
Very good job in following javadocs standard of using creates instead of create:blush: